examples/concepts/access-protection/src/web/guessSecret.ts

```diff
@@ -3,7 +3,9 @@ import checkSecret from '../game/checkSecret';
 
 export default async function guessSecret(secret: string): Promise<string>
```

Comment:
```suggestion
export default async function guessSecret(secret: number): Promise<string>
```

Is this a good idea? It feels more logical to me. The only consequence is that we have to change the requests from GET to POST so we can provide the correct content type.